test/b2gedmntuples_cfg.py

```diff
@@ -768,7 +768,7 @@
 process.boostedJetUserDataAK8 = cms.EDProducer(
     'BoostedJetToolboxUserData',
     jetLabel  = cms.InputTag('jetUserDataAK8'),
-    puppiSDjetLabel = cms.InputTag('packedPatJetsAK8PFPuppiSoftDrop'),
+    puppiSDjetLabel = cms.InputTag('slimmedJetsAK8'),
```

Comment:
Dear Devdatta,
This is for uncorrected subjets, but the corrected ones are very similar.
Yes, some subjets seems to be missing after reclustering, one needs to study more in detail why exactly.
If the above differences are acceptable, I can switch back to use our collections (to be self consistent). The difference from 1 seems to be symmetric, so I don't think it's a big deal, but we will have to remind ourselves that we differ here slightly from the POG recipe. Does it affect approvals?
So should I switch back to our collections? If yes, I make a new commit.